docker-compose-sqlserver.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.1'
+
+services:
+   sqlserver:
+     container_name: SQLServer
+     image: microsoft/mssql-server-linux:latest
+     ports:
+       - "1433"
+     environment:
+       WEB_CONSOLE: "true"
+       DBCA_TOTAL_MEMORY: 3500
```

Comment:
This "DBCA_TOTAL_MEMORY" parameter shows initial or max value that sql server will allocate? 
I would prefer to lower that value, because of TeamCity Cloud agents not always have a lot of memory.

Reply:
I just removed these Oracle-specific Docker options, but SQL Server Docker images officially requires  at least 3.25 GB of RAM. https://hub.docker.com/r/microsoft/mssql-server-linux/